backend/firebaseFunctions.js

```diff
@@ -189,7 +189,10 @@ export async function getFilterCircles() {
 			iconColor: circleData.iconColor,
 			title: circleData.title,
 			color: circleData.color,
-			icon: circleData.icon
+			icon: circleData.icon,
+			description: circleData.description,
+			iconColor: circleData.iconColor,
+			membersList: makeCircleUserList(circle)
```

Comment:
rather than doing this everytime we get the filter circles, we should perform this operation in a `useEffect` in `circleSettings.js`